_posts/en/newsletters/2021-03-03-newsletter.md

```diff
@@ -0,0 +1,158 @@
+  include a link to the documentation on [ReadTheDocs.io][hwi rtd], new
+  and updated [examples][hwi examples], and a new [policy][hwi policy]
+  that describes the criteria new devices must meet for HWI to consider
+  supporting them.
+
+- [Rust Bitcoin #573][] adds a new method
+  `SigHashType::from_u32_standard` that ensures the provided sighash
+  byte is one of [standard values][sighash types] that Bitcoin Core will
+  relay and mine by default.  Each signature's sighash byte indicates
+  what parts of the transaction need to be signed.  Bitcoin's consensus
+  rules dictate that non-standard sighash values are treated as
+  equivalent to `SIGHASH_ALL`, but the fact that they aren't relayed or
+  mined by default can theoretically be used to trick software using
+  offchain commitments into accepting an unenforceable payment.
+  Developers of such software using Rust-Bitcoin may which to switch to
+  this new method from the `SigHashType::from_u32` method that accepts
+  any consensus-valid sighash byte.
+
+- [BIPs #1069][] updates [BIP8][] to allow for a configurable activation threshold
+  and to include 90% as a recommendation, down from 95% previously, based on the
+  most recent [taproot activation discussion][news137 taproot activation].
+
+{% include references.md %}
+{% include linkers/issues.md issues="16546,573,791,794,413,469,463,464,471,468,466,1069,672" %}
+[voegtlin bip70 alt]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-February/018443.html
+[lnurl]: https://github.com/fiatjaf/lnurl-rfc
+[hill scheme]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-February/018446.html
+[kozlik scheme]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-February/018448.html
+[le guilly post]: https://mailmanlists.org/pipermail/dlc-dev/2021-February/000020.html
+[dlcv0 fraud proofs]: https://github.com/discreetlogcontracts/dlcspecs/blob/master/v0Milestone.md#simple-fraud-proofs-in-progress
+[hwi old process]: https://github.com/bitcoin-core/HWI/blob/7b34fc72c5b2c5af216d8b8d5cd2d2c92b6d2457/docs/examples/bitcoin-core-usage.rst
+[hwi release]: /en/newsletters/2019/05/07/#basic-hardware-signer-support-through-independent-tool
+[hwi new process]: https://github.com/bitcoin/bitcoin/blob/master/doc/external-signer.md
+[signer gui]: https://github.com/bitcoin-core/gui/pull/4
+[hwi rtd]: https://hwi.readthedocs.io/en/latest/?badge=latest
+[hwi examples]: https://hwi.readthedocs.io/en/latest/examples/index.html
+[hwi policy]: https://hwi.readthedocs.io/en/latest/devices/index.html#support-policy
+[X.509 certificates]: https://en.wikipedia.org/wiki/X.509
+[news137 taproot activation]: /en/newsletters/2021/02/24/#taproot-activation-discussion
+[sighash types]: https://btcinformation.org/en/developer-guide#signature-hash-types
+[news135 blocksource]: /en/newsletters/2021/02/10/#rust-lightning-774
+[0.19.0 segwit]: https://bitcoincore.org/en/releases/0.19.0.1/#mempool-and-transaction-relay
```

Comment:
Interesting, this link loses the anchor with the tor browser, which converts the url to e.g. http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/en/releases/0.19.0.1/ ...probably an edge case not worth bothering about but https://github.com/bitcoin/bitcoin/pull/15846 works equally well (since GitHub doesn't provide an onion version)

Reply:
If you can verify that it's something about BitcoinCore.org site specifically (not a problem common to all sites with a `http-equiv="onion-location"` tag), please open a bug on the BitcoinCore.org issue tracker and I'll look into it.